packages/client/hmi-client/src/utils/pivot.ts

```diff
@@ -79,10 +79,35 @@ const pivotAxes = (data: any[], rowDimensions: string[], colDimensions: string[]
 	return { colAxis, rowAxis };
 };
 
+// Creates a M x 1 matrix where
+// M =  cardinality(rowDimensions[0]) * cardinality(rowDimensions[1]) * ... * cardinality(rowDimensions[m])
+export const createStateMatrix = (data: any[]) => {
+	const rows: any[] = [];
+
```

Comment:
Are we able to type this (no any)?

Reply:
I think something is a little off here, we should be able to just use `createMatrx` to create a 1D matrix. 

Also utilities should not know anything about "states" or "transitions" - this is supposed to be a generalized pivot-table function.

Reply:
`createMatrix` ends up giving a 2D matrix for states, I'll double check why that is.

Reply:
Or rather it should create a 2D-matrix with one row in it.